### ISSUE TITLE
I hope you will license it/optionally licence it under something not so crude

### ISSUE THREAD

**Issue by madmanpierre** [danpalmer/jquery.complexify.js#34]
We unfortunately cannot use your library due to the 'unacceptable obnoxious clauses' and 'unacceptable obnoxious name' of your current license.
My staff really wanted to use it until they saw how it was licensed, but they acknowledge we cannot possibly use it under such a crude and rude license.
I understand some people will find it funny/cute. And I realize that some people are likely to respond or think about responding to me using the crude word in the license. 
But I am asking you politely to consider licensing it or multi-licensing so that people like us can also use your very nice otherwise library.

**Comment by msigley** [comment]
The WTFPL doesn't even require you to include a reference to the license or a copy of the license itself, it literally means "do whatever you want with it". Pretty much the most loose license in existence short of no license at all.
http://www.wtfpl.net/about/

I mean it could be licensed under the Beerware license after all.
https://en.m.wikipedia.org/wiki/Beerware

-----Original Message-----

**Comment by danpalmer** [comment]
Hi @madmanpierre, thanks for your comments. I'd like to try and understand exactly what you find crude and rude about the licence?

The reasons that I chose the licence are quite simple - I don't want to restrict people from using this as they see fit, however I would prefer people to not re-release derivative works under the same name. This is precisely what the WTF Public Licence grants.

I realise that the licence contains a swear word, but I don't believe the wording is particularly exclusionary, and in terms of the _name_, I would consider "WTF" to be a more common by itself, than its expansion. In addition, the swearing is not directed at a person, or the reader, and is not an attack in any way.

@msigley thanks for your idea - the Beerware licence would be similar, however at the time of publishing this library I did not drink alcohol, so would not have been appropriate then, and I don't see the need to complicate the terms now.
